.env.template

```diff
@@ -143,12 +143,11 @@ GDS_BFF_LOG_LEVEL=info
 GDS_BFF_CONSOLE_LOG=true
 GDS_BFF_ALLOW_ORIGINS="http://localhost:3000,http://localhost:3003"
 GDS_BFF_COOKIE_DOMAIN=localhost
+GDS_BFF_LOGIN_URL=
+GDS_BFF_REGISTER_URL=
 
 GDS_BFF_SERVE_DOCS=true
 
-GDS_BFF_AUTH0_DOMAIN=
```

Comment:
Is `GDS_BFF_AUTH0_DOMAIN` still needed? It looks like we still have it in the docker-compose file

Reply:
Whoops, thanks for catching this, it's a bit of a cognitive load to keep the configuration values in sync across the three different places!